sql/src/main/java/org/apache/druid/sql/http/SqlQuery.java

```diff
@@ -28,13 +28,17 @@
 import org.apache.calcite.avatica.remote.TypedValue;
 import org.apache.druid.java.util.common.ISE;
 import org.apache.druid.query.QueryContext;
+import org.apache.druid.query.http.ClientSqlQuery;
 
 import javax.annotation.Nullable;
 import java.util.List;
 import java.util.Map;
 import java.util.Objects;
 import java.util.stream.Collectors;
 
+/**
+ * @see ClientSqlQuery for the equivalent POJO class for use on the client side.
+ */
 public class SqlQuery
```

Comment:
Can we have this class extend `ClientSqlQuery`?
Alternatively, maybe we could move this class itself to processing module. Is any field in this class stopping us from doing that?